docs/dev/build-instructions/README.md

```diff
@@ -1,10 +1,14 @@
-﻿# Build Mumble
```

Comment:
What is this change about? And what changed at all? :eyes:

Reply:
That's what I thought as well but if that was the case it is really odd that only this single line is affected instead of the entire document :thinking:

Reply:
Is there a way to revert it?

Reply:
You could reset your commits and then do a `git add -p` on the file and don't select this change. Then commit the rest and do a `git restore` on it

Reply:
Thank you, always something new to learn with git :smile:.